xbmc/pvr/recordings/PVRRecording.h

```diff
@@ -159,6 +163,12 @@ namespace PVR
      */
     CBookmark GetResumePoint() const override;
 
+    /*!
+     * @brief Update this recording's size. The value will be obtained from the backend if it supports server-side size retrieval.
```

Comment:
line too long

Reply:
Comments are not subject to this guideline.

Reply:
Hmm. Not sure. What does clang-format say to this?

Reply:
It didn't flag it when I ran it, it was left as is.

Reply:
Okay, then.